src/index.html

```diff
@@ -11,6 +11,40 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+     action="#"
+     class="first_search"
+     method="get"
+    >
+      <input
+        class="first_search_input"
+        type="search"
+        name="bigSearch"
+        data-qa="big"
+        placeholder="Try &#34;Los Angeles&#34;"
+      >
+      <button class="searcher_img">
+        <img src="./images/Search.svg" alt="img">
+      </button>
+    </form>
+    <form 
+      action="#"
+      class="second_search"
+      method="get">
+      <input
```

Comment:
p5 [Checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)

Reply:
not fixed